utilities/cwp/custom-war-packager.inc

```diff
@@ -0,0 +1,137 @@
+# function to download the CWP jar
```

Comment:
Why don't we use the CWP Docker image. Since the framework is Docker-focused, I do not see much reason to use a non-dockerized version here

Reply:
Are there versions/incrementals for the Docker image?

Reply:
@oleg-nenashev do you mean to clone the repo and build the docker image?
Let's do it in a follow-up